guava/src/com/google/common/hash/PrimitiveSink.java

```diff
@@ -42,7 +47,7 @@
    * @param bytes a byte array
    * @return this instance
    */
-  PrimitiveSink putBytes(byte[] bytes);
+  PrimitiveSink putBytes(byte @MinLen(1)[] bytes);
```

Comment:
Why `@MinLen(1)`?

Reply:
To be more explicit: is there reason to say that implementations of this method are allowed to fail if an empty array is passed as an argument?